.github/workflows/tracing-runtime.yml

```diff
@@ -0,0 +1,55 @@
+name: Build and publish evm tracing runtime
+on:
+  release:
+    types: [published]
+  # Allow manual trigger for old releases
+  workflow_dispatch:
```

Comment:
This is useful for the current version. We'll need to create a branch from old tag and include tracing feature and then trigger this action to build tracing runtime and push to specified tag.